<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScrapeImagesSettings</class>
 <widget class="QWidget" name="ScrapeImagesSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>1394</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="scrapeImagesGroupBox">
     <property name="title">
      <string>Scrape Images</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QLineEdit" name="downloadImagesDirectoryLineEdit"/>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="downloadImagesToDirectoryCheckBox">
        <property name="text">
         <string>Download images to directory</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="browsePushButton">
        <property name="text">
         <string>Browse ...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>529</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>downloadImagesToDirectoryCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>downloadImagesDirectoryLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
